packages/sqflite/tizen/src/constants.h

```diff
@@ -0,0 +1,66 @@
+#ifndef CONSTANTS_H
+#define CONSTANTS_H
+
+#include <string>
+
+const std::string kPluginKey = "com.tekartik.sqflite";
+const std::string kMethodGetPlatformVersion = "getPlatformVersion";
```

Comment:
Is this method implementation missing? 
`kMethodGetPlatformVersion` seems to have been deprecated in v1.3.2.